llvm/test/Analysis/ScalarEvolution/no-wrap-add-exprs.ll

```diff
@@ -362,3 +362,20 @@ loop:
 exit2:
   ret i1 false
 }
+
+
+define void @dereferenceable_arg(ptr dereferenceable(128) %len_addr, ptr dereferenceable(128) align(8) %len_addr2) {
```

Comment:
Can you please also add a test using dereferenceable_or_null?

Reply:
Done